frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -573,6 +637,29 @@ export function ContainerInfo(props: ContainerInfoProps) {
       env[envVar.name] = value;
     });
 
+    function preparePortForwardComponent(containerPort: number) {
+      if (!helpers.isElectron()) {
+        return null;
+      }
+      return !portForwardInitiated ? (
+        <Button onClick={() => handlePortForward(namespace, podname, containerPort)}>
+          Port Forward
```

Comment:
All buttons like this this need to be translatable.

Reply:
updated